pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc-view-results.adoc

```diff
@@ -0,0 +1,10 @@
+[id='view-results_{context}']
+
+.Viewing search results
+The search results are presented in a table organized into columns with the following headings:
+
+* `Title` - the module title
+* `Published` - the publication status, `Published` or `Not published`
+* `Draft uploaded` - the date the draft was uploaded
+* `Module Type` - the module type, `Concept`, `Procedure`, or `Reference`
+* A sorting button allows you to quickly change the sorting order of search results from A-Z to Z-A.
```

Comment:
Again, just a note that the sort button only applies to the next search, not the current set of results.